zonal_datacube/geo_utils.py

```diff
@@ -0,0 +1,52 @@
+import math
+
+import numpy as np
+
+
+def get_hectare_area(lat: float, pixel_width: float) -> float:
```

Comment:
I was surprised to find this. None of our common gis packages do this? Or, is it just not worth the dependency? Just curious.